native/jest.setup.js

```diff
@@ -12,6 +12,9 @@ if (typeof window !== 'object') {
 global.fetch = require('jest-fetch-mock')
 jest.mock('rn-fetch-blob')
 
+// FormData is not part of the jest react-native preset: https://github.com/jefflau/jest-fetch-mock/issues/23
+global.FormData = require('testing/FormDataMock.js')
```

Comment:
Do you know why this was not needed in the api-client? or was it?

Reply:
No, it was not. It was not needed because this is somehow a problem with the jest `react-native preset` as mentioned in the comment. Without it everything works fine.